cinema/urls.py

```diff
@@ -18,6 +18,8 @@
 router.register("movie_sessions", MovieSessionViewSet)
 router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [
+    path("", include(router.urls))
+]
```

Comment:
No need to include if basepath is not provided
If you change this change to 
urlpatterns = router.urls

Reply:
Fixed